FLEDGE.md

```diff
@@ -108,6 +109,8 @@ The browser will remain in an interest group for only a limited amount of time.
 
 #### 1.2 Interest Group Attributes
 
+The `priority` is used to select which interest groups are shown when the number of interest groups are limited by the `perBuyerGroupLimits` attribute of the auction config. If not specified, a `priority` of `0.0` is assigned.
```

Comment:
"which interest groups are _shown_"?  It's which groups participate in an auction (as you describe below), so "shown" seems wrong here.

There's no special meaning to the numerical values, right?  That is, 0.0 here is just a neutral default, and it's OK to use positive and negative numbers?

Reply:
Done